nilearn/plotting/displays.py

```diff
@@ -884,15 +933,28 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
                 kwargs['color'] = 'k'
 
         bg_color = ('k' if self._black_bg else 'w')
+
+        # Cleanup scale bar's kwargs
+        scale_bar_kwargs = {
+            key: kwargs.pop(key) for key in ['scale_width', 'units']
+            if key in kwargs}
```

Comment:
These arguments should be added explicitly to the function signature, rather than via the kwargs.